dash-renderer/package.json

```diff
@@ -31,15 +32,16 @@
     "clean-webpack-plugin": "^0.1.19",
     "cookie": "^0.3.1",
     "cross-env": "^4.0.0",
+    "check-prop-types": "1.1.2",
     "dependency-graph": "^0.5.0",
     "json-loader": "^0.5.4",
-    "lodash": "^4.17.11",
-    "prop-types": "^15.6.2",
+    "lodash": "^4.17.15",
+    "prop-types": "15.7.2",
     "query-string": "^4.3.2",
     "radium": "^0.22.1",
     "ramda": "^0.26.1",
-    "react": "^16.2.0",
-    "react-dom": "^16.2.0",
+    "react": "16.8.6",
```

Comment:
to avoid surprise upgrade
if we leave it like before, react will bump to 16.9.0, and we end up with a lot of deprecated warnings as notice for react 17.x migration

Reply:
While at it, I'd suggest we move all `dependencies` except
* check-prop-types
* prop-types
* react
* react-dom

to `devDependencies` to match usage (packaged vs. unpackaged deps)